tests/playwright-test/test-extend.spec.ts

```diff
@@ -225,3 +225,43 @@ test('test._extendTest should print nice message when used as extend', async ({
   expect(result.passed).toBe(0);
   expect(result.output).toContain('Did you mean to call test.extend() with fixtures instead?');
 });
+
+test('fixture options should ignore undefined value', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'playwright.config.ts': `
+      module.exports = {
+        use: { option: undefined },
+      };
+    `,
+    'a.test.js': `
+      const test = pwt.test.extend({ option: [ 'default', { option: true } ] });
+      test('test1', async ({ option }) => {
+        console.log('test1-' + option);
+      });
+
+      test.describe('', () => {
+        test.use({ option: 'foo' });
+        test('test2', async ({ option }) => {
+          console.log('test2-' + option);
+        });
+
+        test.describe('', () => {
+          test.use({ option: undefined });
```

Comment:
Shouldn't we throw in this case? I would expect such config to delete previously set value.